tdrs-backend/tdpservice/search_indexes/migrations/0028_auto_20240429_1805.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.15 on 2024-04-29 18:05
+
+from django.db import migrations, models
```

Comment:
Can we name this migration file according to `migration-best-practices.md` in `docs/Technical-Documentation`?